packages/core/next.config.js

```diff
@@ -40,6 +41,16 @@ const nextConfig = {
         }
       })
 
+    // Make css order works as expected in Next 13. See https://github.com/vercel/next.js/issues/51030#issuecomment-2005254907
+    config.plugins.push(
```

Comment:
note: Nice! There are many recent issues related to it, let's keep in mind to see if there is an update regarding it later.